releasenotes/notes/profiler-gevent-warns-on-late-patch-3f4f897c8904591c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    The `ddtrace.profiling.auto` module will warn users if gevent monkey
+    patching is done after the profiler is a
```

Comment:
after the profile is a what??? hahah such a cliff hanger 😆 

```suggestion
    patching is done after the profiler is auto-instrumented.
```
?

Reply:
lol
the perks of being interrupted all the time 😆